actor/src/main/scala/com/evolutiongaming/akkaeffect/util/Serially.scala

```diff
@@ -57,7 +57,7 @@ private[akkaeffect] object Serially {
               (S.Active, start(s.value, t))
             case s: S.Active =>
               val task = (a: A) => for {
-                a <- s.task(a)
+                a <- Async[F].defer(s.task(a))
```

Comment:
could you please defer everything that goes after (a: A) =>

Reply:
it changes signature from F[A] to F[F[A]], do you see any benefits of doing that?

Reply:
Done

Reply:
nevermind, got you